diff --git a/packages/kit/src/runtime/client/client.js b/packages/kit/src/runtime/client/client.js
index f866a04c1b0..1759dd6b8e8 100644
--- a/packages/kit/src/runtime/client/client.js
+++ b/packages/kit/src/runtime/client/client.js
@@ -1251,7 +1251,7 @@ export function create_client({ target, base, trailing_slash }) {
 				// See:
 				// - https://github.com/sveltejs/kit/issues/4045
 				// - https://github.com/sveltejs/kit/issues/5725
-				if (!is_svg_a_element && !(url.protocol === 'https:' || url.protocol === 'http:')) return;
+				if (!is_svg_a_element && !(url.protocol === 'https:' || url.protocol === 'http:' || url.protocol === 'tauri:')) return;

 				// Ignore if tag has
 				// 1. 'download' attribute